documentation/modules/creating-migration-plan-2-6-3.adoc

```diff
@@ -56,6 +56,12 @@ To specify a different root device, in the *Settings* section, click the Edit ic
 +
 If the conversion fails because the boot device provided is incorrect, it is possible to get the correct information by looking at the conversion pod logs.
 
+* *{ocp-short} transfer network*: You can configure an {ocp-short} network in the ocp-short} web console by clicking *Networking > NetworkAttachmentDefinitions*.
```

Comment:
can we tidy up the naming convention, please?
it does not look good on URLs for MTV 2.7 to see 2.6.3 in the URL

Reply:
Good point. Fixed.